storybook/stories/index.js

```diff
@@ -110,34 +178,149 @@ storiesOf('Components/Fee Slider', module)
     />
   ))
 
-storiesOf('Components/Fetching Indicators', module)
+storiesOf('Components/Labels', module)
   .addParameters({
     options: {
       hierarchySeparator: /\//,
       hierarchyRootSeparator: /\|/,
     },
   })
-  .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('FetchingIndicator FETCHING', () => (
-    <FetchingIndicator status='FETCHING' />
+  .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
+  .add('Labels', () => (
```

Comment:
All variants with labels must be merged into one story (one screen)

Reply:
I have left Labels and Labelitems separated